modules/workergroup/clusternetworks.tf

```diff
@@ -0,0 +1,43 @@
+# Copyright 2022, Oracle Corporation and/or affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl
+
+# Dynamic resource block for Cluster Network groups defined in worker_groups
+resource "oci_core_cluster_network" "cluster_networks" {
+  for_each = {
+    for k, i in var.worker_groups : k => i
+    if coalesce(lookup(i, "mode", "unknown"), local.mode) == "cluster-network"
+    && lookup(i, "enabled", var.enabled)
+  }
+
+  compartment_id = lookup(each.value, "compartment_id", local.compartment_id)
+  display_name   = join("-", compact([lookup(each.value, "label_prefix", var.label_prefix), each.key]))
+  defined_tags   = merge(coalesce(local.defined_tags, {}), lookup(each.value, "defined_tags", {}))
+  freeform_tags  = merge(coalesce(local.freeform_tags, {}), lookup(each.value, "freeform_tags", { oke_group = each.key }))
+
+  instance_pools {
+    instance_configuration_id = oci_core_instance_configuration.instance_configuration[each.key].id
+    display_name              = join("-", compact([lookup(each.value, "label_prefix", var.label_prefix), each.key]))
+    size                      = max(0, lookup(each.value, "size", local.size))
+    defined_tags              = merge(coalesce(local.defined_tags, {}), lookup(each.value, "defined_tags", {}))
+    freeform_tags             = merge(coalesce(local.freeform_tags, {}), lookup(each.value, "freeform_tags", { oke_group = each.key }))
+  }
+
+  placement_configuration {
+    availability_domain = local.ad_number_to_name[lookup(each.value, "placement_ads", local.ad_numbers)[0]]
+    primary_subnet_id   = var.primary_subnet_id
+  }
+
+  lifecycle {
+    ignore_changes = [
+      display_name,
+      time_updated,
```

Comment:
Looks like this field raises this warning

```
╷
│ Warning: Redundant ignore_changes element
│ 
│   on modules/workergroup/clusternetworks.tf line 5, in resource "oci_core_cluster_network" "cluster_networks":
│    5: resource "oci_core_cluster_network" "cluster_networks" {
│ 
│ Adding an attribute name to ignore_changes tells Terraform to ignore future changes to the argument in configuration after the object has been created, retaining the value originally configured.
│ 
│ The attribute time_updated is decided by the provider alone and therefore there can be no configured value to compare with. Including this attribute in ignore_changes has no effect. Remove the attribute
│ from ignore_changes to quiet this warning.
```

Reply:
Fixed; thanks!